<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DWDS Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="vocab_layout">
      <item>
       <widget class="QLabel" name="vocab_label">
        <property name="text">
         <string>Vokabular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vocab_lineEdit">
        <property name="text">
         <string>Buch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="vocab_button_copy">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_grammar" columnstretch="0,1,0">
        <item row="0" column="2">
         <widget class="QPushButton" name="grammar_button_copy">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="grammar_label">
          <property name="text">
           <string>Grammatik</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="grammar_spacer_label">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="grammar_spacer_copy">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QPlainTextEdit" name="grammar_plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_meaning" columnstretch="0,1,0">
        <item row="0" column="0">
         <widget class="QLabel" name="meaning_label">
          <property name="text">
           <string>Bedeutung</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="meaning_button_copy">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="meaning_spacer_label">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="meaning_spacer_copy">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QPlainTextEdit" name="meaning_plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_examples" columnstretch="0,1,0">
        <item row="0" column="0">
         <widget class="QLabel" name="examples_label">
          <property name="text">
           <string>Beispiele</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="examples_slider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="5">
         <widget class="QPlainTextEdit" name="examples_plainTextEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="examples_button_copy">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="examples_spacer_copy">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="4">
         <spacer name="examples_spacer_label">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="examples_label_count">
          <property name="text">
           <string>Count</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_thesaurus">
        <item row="0" column="0">
         <widget class="QLabel" name="thesaurus_label">
          <property name="text">
           <string>Verwandte</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="thesaurus_button_copy">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="thesaurus_spacer_copy">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="thesaurus_spacer_label">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_tags">
          <property name="text">
           <string>Tags</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QPlainTextEdit" name="thesaurus_plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
